.github/workflows/ci.yml

```diff
@@ -551,12 +551,19 @@ jobs:
         uses: actions/setup-python@v5
         with:
           python-version: "3.12"
+      - name: "Setup Python environment"
+        run: |
+          pipx install poetry
+          poetry config virtualenvs.create true --local
+          poetry env use 3.12
+      - name: "Install dependencies"
+        run: "poetry install --no-interaction --no-ansi"
       - name: "Setup environment"
         run: "pip install invoke toml"
       - name: "Build Test Image"
-        run: "invoke dev.build"
+        run: "poetry run invoke dev.build"
```

Comment:
Do we need this step since the next command is not running in docker anymore ?

Reply:
I didn't think about it, I'll test it